README.md

```diff
@@ -39,3 +41,13 @@ To use the plugin you need to register it, like most Storybook plugins. Simply a
 ## Comment formatting
 
 Comments are formatted using the [marked](https://www.npmjs.com/package/marked) package.
+
+## Local CouchDB setup
+
+[Install Apache CouchDB](http://couchdb.apache.org/) by downloading the respective installer for your OS.
+
+Once installation is done, navigate to [http://localhost:5984](http://localhost:5984). Create an admin username and password.
```

Comment:
To make sure that no one can access your local CouchDB setup and destroy things without admin access? In case it gets hijacked via sneaky terminal scripts? I dunno. Just do it.